docs/user-docs/openqa.md

```diff
@@ -0,0 +1,244 @@
+there.
+
+|![Image of the Job Group view](../img/openqa/job-group-view.png)
+|:-:|
+|Image of the Job Group view|
+
+### Product
+
+A _Product_ represents one run of a set of test suites with given parameters.
+The name of the products consists of a couple parts
+It consists of:
+- The name of the tested OS distribution - to run tests on every maintained
+  distribution
+- The version - to allow running the tests on every new release
+- The flavor - to allow testing every flavor of the OS separately
+- The hardware architecture - to run on every supported architecture
+
+Testing TrenchBoot requires to cover all of the supported hardware platforms,
+not only run a single test per hardware architecture. For this reason the
+_Flavor_ field of a _Product_ is used a bit unconventionally. At this moment
+TrenchBoot does not consist of multiple _Flavors_ and the field is used to
+determine:
+- on which hardware platform the tests were performed
+- whether it was installed on a Legacy or an UEFI BIOS
+
+|![Product name made of four sections](../img/openqa/medium-string.png)|
+|:-:|
+|Product name made of four sections|
+
+Clicking on a _Product_ from the main page, or the _Job Group_ details page
+shows the summary of the _Product_ and a list of the _Test Suites_ associated
+with it.
+
+|![Image of a product view with a list of associated tests](../img/openqa/product-view.png)|
+|:-:|
+|Image of a product view with list of tests|
+
+### Test Suite
+
+_Test Suites_ are the essence of the OpenQA web panel. Next to every test suite
+you can see a small colorful circle informing about the status of the
+_Test Suite_. The most common statuses are:
+- Gray - Skipped
+    + The test was decided to not be performed based on the results of
+    other tests, like its dependencies failing
+- Green - Pass
+    + The test passed without any issues
+- Red - Fail
+    + The test encountered a critical issue, the testing is stopped
+- Yellow - Soft Fail
+    + The test encountered a non-critical issue, the testing may continue
+
+By clicking on the status circle icon informing on the status of the
+_Test Suite_, the details of the selected _Test Suite_ can be accessed.
+
+|![Image of the circle to press, which is very small and a bit unintuitive, marked with a red rectangle](../img/openqa/test-details-button.png)|
+|:-:|
+|Image of the circle to press, which is very small and a bit unintuitive, marked with a red rectangle|
+
+When accessing the _Test Suites_ from the _All Tests_ view shown above, you
+can also click on the blue _Test Suite_ name to access the details. On most
+other pages pressing the circle is the only way to do so.
+
+Upon doing so the page of the selected _Test Suite_ run as
+a part of a _Product_ will be shown. It contains a short
+summary of the _Test Suite_ and a tab bar containing
+a couple tabs with more details.
+
+#### Details Tab
+
+The default tab shown when selecting a _Test Suite_ is its
```

Comment:
```suggestion
The default tab shown when selecting a finished _Test Suite_ is its
```
Running ones open in live view and assigned/scheduled ones don't have details.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/55/commits/a818b48fd712a724973cbe959b1b9894939384de